util/config_file.c

```diff
@@ -790,7 +791,8 @@ int config_set_option(struct config_file* cfg, const char* opt,
 		}
 		oi[cfg->num_out_ifs++] = d;
 		cfg->out_ifs = oi;
-	} else {
+	} else S_YNO("local_data_do_ede:", local_data_do_ede)
+	else {
```

Comment:
Part of the not so simple EDE cases?

```suggestion
	} else  {
```

Reply:
resolved in 71e7bbf